gilded-rose-kata/src/main/kotlin/Item.kt

```diff
@@ -1,8 +0,0 @@
-data class Item(
```

Comment:
Originally, the `Item` class should not be modified. Let's ignore that for this first implementation, but keep it in mind for the next one using TDD. See [restrictions](kotlin-gilded-rosa-kata) for more details.